CHANGELOG.md

```diff
@@ -1 +1,68 @@
 # Changelog
+
+Items starting with `DEPRECATE` are important deprecation notices. For more
+information on the list of deprecated flags and APIs, have a look at
+https://docs.docker.com/engine/deprecated/ where you can find target removal dates.
+
+## 18.02.0-ce (2018-02-DD)
+
+### Builder
+
+* Install-containerd-static: use netgo buildtag [moby/moby#35753](https://github.com/moby/moby/pull/35753)
++ Add dockerfile for building on non-amd64 platforms [docker/cli#816](https://github.com/docker/cli/pull/816)
```

Comment:
I would either remove it or move it to client, it's not really related to the builder